### ISSUE TITLE
Tracking Auras by id

### ISSUE THREAD

**Issue by shocker13** [XiconQoo/Gladdy-TBC#64]
Greetings. For some reason Gladdy is tracking auras only by name which leads to showing unwanted auras. For example silence from "unstable affliction" has the same name with "unstable affliction" dot and if you're playing with warlock, all arena frames are perma blocked by the dot. 
![qUUtxLH](https://user-images.githubusercontent.com/18690368/186328826-7f824bd1-91ab-4d55-a154-77cdacdfd0a1.png)
Another example is ferals "berserk" that has the same name with "berserk" weapon enchant.
![STkMz24zgs](https://user-images.githubusercontent.com/18690368/186329423-dbcd264f-098d-4656-bc57-e7e3babf4e2f.png)
In other words it would be great to track auras only by id. Cheers.

**Comment by XiconQoo** [comment]
Please download latest Gladdy. It's fixed there

**Comment by shocker13** [comment]
It's fixed indeed. But it didn't solve the problem itself. "Wyvern Sting" poison has the same name as the sleep effect and etc, etc. The only way to prevent this type of issues is to track auras by id.

**Comment by shocker13** [comment]
It's fixed indeed. Nice work.
